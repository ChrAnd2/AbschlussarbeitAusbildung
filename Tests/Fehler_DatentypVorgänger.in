//**********************************************************
//+ Fehler_DatentypVorgänger - Character in Vorgänger
//**********************************************************
//Vorgangsnummer; Vorgangsbezeichnung; Dauer; Vorgänger; Nachfolger
1; A; 21; -; 2,3
2; B; 5; 1; 4
3; C; 7; A; 6
4; D; 3; 2; 5,6
5; E; 5; 4; 7
6; F; 8; 3,4; 7
7; G; 2; 5,6; -