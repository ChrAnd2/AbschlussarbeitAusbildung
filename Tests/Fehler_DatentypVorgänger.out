//**********************************************************
//+ Fehler_DatentypVorgänger - Character in Vorgänger
//**********************************************************
//Vorgangsnummer; Vorgangsbezeichnung; Dauer; Vorgänger; Nachfolger
1; A; 21; -; 2,3
2; B; 5; 1; 4
3; C; 7; A; 6
4; D; 3; 2; 5,6
5; E; 5; 4; 7
6; F; 8; 3,4; 7
7; G; 2; 5,6; -


*************************************
Es ist ein Fehler aufgetreten.
*************************************
EingabeException: Vorgangsnummer eines Vorgängers des Vorgangs mit der Vorgangsnummer 3 ist keine ganzzahlige Zahl.
